Mobile/lib/presentation/pages/questions.detail/questions_detail.dart

```diff
@@ -0,0 +1,235 @@
+        dislikes: ["1,23"],
+        author: "Doju",
+        anonymous: false,
+        question: "2",
+        createdAt: DateTime.now(),
+        updatedAt: DateTime.now()),
+    Answer(
+        id: "1",
+        text: "I don't know man",
+        image: "",
+        likes: ["1", "3"],
+        dislikes: ["1,23"],
+        author: "Doju",
+        anonymous: false,
+        question: "2",
+        createdAt: DateTime.now(),
+        updatedAt: DateTime.now()),
+    Answer(
+        id: "2",
+        text: "I don't know man",
+        image: "",
+        likes: ["1", "3"],
+        dislikes: ["1,23"],
+        author: "Doju",
+        anonymous: false,
+        question: "2",
+        createdAt: DateTime.now(),
+        updatedAt: DateTime.now()),
+    Answer(
+        id: "3",
+        text: "I don't know man",
+        image: "",
+        likes: ["1", "3"],
+        dislikes: ["1,23"],
+        author: "Doju",
+        anonymous: false,
+        question: "2",
+        createdAt: DateTime.now(),
+        updatedAt: DateTime.now()),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    List<Widget> answerWidget = answerList
+        .map((answer) => Row(
+              children: [
+                SizedBox(
+                  width: 10,
+                ),
+                AnswerW(answer: answer),
+              ],
+            ))
+        .toList();
+    return Scaffold(
+      appBar: AppBar(title: Text('question comments')),
+      bottomNavigationBar: BottomAppBar(),
+      body: Column(
+        children: [
+          SizedBox(
+            height: 10,
+          ),
+          QuestionW(question: question),
+          Container(
+            // color: Colors.red,
+            child: Expanded(
+              child: ListView.builder(
+                  itemCount: answerList.length,
+                  itemBuilder: (context, index) {
+                    return Padding(
+                      padding: const EdgeInsets.only(
+                          left: 16, right: 0, top: 0, bottom: 5),
+                      child: answerWidget[index],
+                    );
+                  }),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(16.0, 16.0, 0.0, 8.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                Image.asset('assets/images/user 1.png'),
+                SizedBox(
+                  width: 10.0,
+                ),
+                Expanded(
+                  child: Container(
+                    height: 50,
+                    child: TextField(
+                      decoration: InputDecoration(
+                        suffixIcon: IconButton(
+                            onPressed: () {}, icon: Icon(Icons.send)),
+                        border: InputBorder.none,
+                        hintText: "reply",
+                      ),
+                    ),
+                  ),
```

Comment:
![Screenshot (112)](https://github.com/Merwan-J/AskAnything/assets/105606795/8fdace89-6f4e-4ad7-9a53-df8555a748ac)

when the profile is touched it should have a little menu and let the user choose if he wants to answer as anonymous or not

Reply:
and also give space between the question and answer.

Reply:
check it out esti